src/game.cpp

```diff
@@ -748,14 +799,17 @@ class GameGlobalShaderConstantSetterFactory : public IShaderConstantSetterFactor
 	bool *m_force_fog_off;
 	f32 *m_fog_range;
 	Client *m_client;
+	Camera *m_camera;
 	std::vector<GameGlobalShaderConstantSetter *> created_nosky;
+	std::vector<GameGlobalShaderConstantSetter *> created_camera;
 public:
 	GameGlobalShaderConstantSetterFactory(bool *force_fog_off,
 			f32 *fog_range, Client *client) :
 		m_sky(NULL),
 		m_force_fog_off(force_fog_off),
 		m_fog_range(fog_range),
-		m_client(client)
+		m_client(client),
+		m_camera(NULL)
```

Comment:
Well the code around this area already used NULL, should I change all of the nearby NULLs to nullptr also for consistency?